### ISSUE TITLE
Stylesheet `twocolumn` is not found

### ISSUE THREAD

**Issue by lornajane** [rst2pdf/rst2pdf#1159]
**Description of problem**
We recently added the twocolumn stylesheet back into the main codebase #1126. However I don't seem to be able to run `rst2pdf -s twcolumn myfile.rst` - the stylesheet isn't found and my document isn't in two columns.

I notice that the test `test_stylesheet_includes` passes, but it emits a warning into its log and looking at the PDF, it's not in two columns (the style does what it sounds like it would).

I worked around my problem by grabbing the old `twocolumn.yaml` file from the git history and using that locally.

🖥 **Versions**

`python -V`
Python 3.10.8

`pip freeze | grep rst2pdf`
`-e git+ssh://git@github.com/rst2pdf/rst2pdf.git@9bb7e8ee8b597a7cd40f36e89fc10f76c0d0a5ca#egg=rst2pdf` <-- what even is that?

`pip freeze | grep reportlab`
reportlab==3.6.12

Which operating system are you using?
Fedora

**Comment by lornajane** [comment]
In case anyone else needs the file to work around the same problem, here it is

```
pageSetup:
  firstTemplate: twoColumn
  margin-left: 1cm
  margin-right: 1cm
pageTemplates:
  twoColumn:
    frames:
    - - 0cm
      - 0cm
      - 49%
      - 100%
    - - 51%
      - 0cm
      - 49%
      - 100%
    showFooter: true
    showHeader: true

```

**Comment by akrabat** [comment]
Historical context:

* Originally, there was a page template `twoColumn` that was in the default stylesheet and there was also a `twocolumn` style sheet that set `firstTemplate` to `twoColumn`.
* In the stylesheet refresh PR, the `twoColumn` page template was moved to the `twocolumn` style sheet and removed from the default stylesheet.
* This caused a Sphinx issue, which I fixed by moving the `twoColumn` page template back to the default stylesheet and removing the `twocolumn` style sheet. 
* Removing the `twocolumn` stylesheet was a mistake, so the fix is to restore `twocolumn.yaml` to be as it was before the stylesheet refresh PR.
